mmv1/products/bigquery/DatasetAccess.yaml

```diff
@@ -301,3 +303,47 @@ properties:
           A-Z), numbers (0-9), or underscores (_). The maximum length
           is 256 characters.
         required: true
+  - name: 'condition'
+    type: NestedObject
+    description: |
+      Condition for the binding. If CEL expression in this field is true, this
+      access binding will be considered.
+    properties:
+      - name: expression
+        type: String
+        required: true
+        description: |
+          Textual representation of an expression in Common Expression Language syntax.
+      - name: title
+        type: String
+        description: |
+          Title for the expression, i.e. a short string describing its purpose.
+          This can be used e.g. in UIs which allow to enter the expression.
+      - name: description
+        type: String
+        description: |
+          Description of the expression. This is a longer text which describes the expression,
+          e.g. when hovered over it in a UI.
+      - name: location
+        type: String
+        description: |
+          String indicating the location of the expression for error reporting, e.g. a file
+          name and a position in the file.
+  - name: 'accessPolicyVersion'
```

Comment:
I don't believe this should be configurable, could you confirm with the API team?

Reply:
sure, will reach out to them